### PR TITLE
v5(array functions)

### PR DESCRIPTION
**Pull request by USERSATOSHI**
## Type
- [ ] Bug Fix
- [x] Functions: \<Function Name>
- [ ] Callbacks: \<Callback Name>
- [ ] Handlers: \<Handler Type>
- [ ] Others: \______

Dependencies (Third Party Modules) needed: <Name | Github Link> (Maximum: 20MB~ size)

Want a credit? Discord tag or other social media link: <DiscordTag | OtherSocialMediaLink>

Referenced Issue: #NaN (Answer for an issue, if any)

## Description
Pull Request Description Here

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,6 +50,8 @@
   },
   "devDependencies": {
     "@akarui/aoi.music": "^1.2.5",
+    "discord-canvas": "^1.4.1",
```
Comment (Leref) [inline package.json]: ?

File: package.json [hunk 2]
```diff
@@ -50,6 +50,8 @@
...
+    "moment": "^2.29.3",
```
Comment (Leref) [inline package.json]: Isn't there a better approach?

Comment (USERSATOSHI) [reply package.json]: oh wait that was for something i was testing on my bot whoops

Comment (USERSATOSHI) [reply package.json]: tho i am pretty sure moment is used by $formatDate

Comment (Leref) [reply package.json]: I would check again.

File: package-lock.json [hunk 3]
```diff
@@ -15,6 +15,8 @@
       },
       "devDependencies": {
         "@akarui/aoi.music": "^1.2.5",
+        "discord-canvas": "^1.4.1",
```
Comment (Leref) [inline package-lock.json]: please revert changes here.

**Review by Leref — CHANGES_REQUESTED** [review]

**Comment by Leref** [comment]
Also resolve the interpreter conflict.

**Pull request closed without merging**